core/trie/node_test.go

```diff
@@ -22,6 +22,7 @@ func TestNodeHash(t *testing.T) {
 	node := trie.Node{
 		Value: new(felt.Felt).SetBytes(valueBytes),
 	}
+	// it's 6 because 2^5 is 32 which is not sufficient to store 42 ?
```

Comment:
I believe 5 bits are enough to store 42 as well, this was just a testcase I took from pathfinder repository.